src/main/java/com/hcl/appscan/jenkins/plugin/scanners/DynamicAnalyzer.java

```diff
@@ -245,7 +247,9 @@ public Map<String, String> getProperties(VariableResolver<String> resolver) thro
 		properties.put(LOGIN_TYPE,m_loginType);
 		properties.put(SCAN_TYPE, m_scanType);
 		properties.put(TEST_OPTIMIZATION_LEVEL, m_optimization);
-		properties.put(PRESENCE_ID, m_presenceId);
+		if (!m_presenceId.equals(EMPTY)) {
+				properties.put(PRESENCE_ID, m_presenceId);
```

Comment:
why this check has to be in jenkins and not in SDK ?

Reply:
Kripa, I think, It is good if we filter the unwanted parameters in initial stage instead of doing it later.